sites/03-moist_bubble.md

```diff
@@ -24,46 +27,57 @@ S(h) =
 \end{align}
 \right.
 ```
+
 and $h\equiv(\|\vec{x}'\|/R-f_s)/(1-f_s)$, we choose $f_s=0.8$ and bubble radius $R = 800$.
-For further information about the example, read section 3.4 in
-[Frey et al (2023)](https://doi.org/10.1016/j.jcpx.2023.100136).
+For further information about the example, read section 3.4 in [Frey et al (2023)](https://doi.org/10.1016/j.jcpx.2023.100136).
 
 We provide you with a Python script that creates the gridded input fields. After following the instructions on
 [how to load the Python virtual environment](02-setup_environment.md#how-to-load-the-python-virtual-environment),
-you can run the script
-[input/write_moist_setup.py](../input/write_moist_setup.py) which creates a file called
-`moist_<nx>x<ny>x<nz>.nc`, where `<nx>`, `<ny>` and `<nz>` are replaced
-by the number of grid cells per dimension (default: `nx = ny = nz = 64`). You can call the script
-with the `--help` argument to get further information.
+you can run the script [input/write_moist_setup.py](../input/write_moist_setup.py) using the following command:
+
+```bash
+python3 write_moist_setup.py
```

Comment:
Simply `python`  should be enough. However, we can keep `python3` if you prefer.

Reply:
Force of habit, I'm happy either way